lib/src/main/java/graphql/nadel/definition/hydration/NadelDefaultHydrationDefinition.kt

```diff
@@ -34,11 +32,9 @@ class NadelDefaultHydrationDefinition(
                     "How to identify matching results"
                     identifiedBy: String! = "id"
                     "The batch size"
-                    batchSize: Int
+                    batchSize: Int! = 200
```

Comment:
Updated to match normal `@hydrated` definition.

Not sure why it was different to begin with.